Marlin/src/lcd/menu/menu_bed_corners.cpp

```diff
@@ -121,7 +122,7 @@ static inline void _lcd_level_bed_corners_get_next_position() {
 
       case 2:
         // Determine which edge to probe for 3rd point
-        current_position.set(X_CENTER, Y_CENTER);
+        current_position.set(lf.x + (rb.x - lf.x) / 2, lf.y + (rb.y - lf.y) / 2);
```

Comment:
I had thought about doing this change as well, but after some thought I wasn't sure that it really made sense. 

The probe inset positions are to ensure that the probe can hit the bed without hitting something like a bed clamp. The center positions don't really have that problem. Additionally, the way I was thinking was that the user would set it up to be as reasonably close to the first 2 edge screws as they can probe, and the 3rd screw is likely along the centerline of the bed, which is where the _CENTER commands would put the probe. 

Is your idea here to split the difference so that one the probe measurement doesn't favor one side or the other if the offsets are uneven? I don't necessarily disagree with doing this change, but I wanted to add perspective about why I left it as _CENTER.

Reply:
Under the current code the point will always line up with the center. Someone would need to hack up the code to change it. Generally, the point exactly between the other two corners will be what is wanted, even if there is some kind of offset added later.